lib/process_settings/file_monitor.rb

```diff
@@ -24,6 +26,31 @@ def initialize(file_path, logger:, environment: nil)
       start_internal(enable_listen_thread?(environment))
     end
 
+    def start_watchdog_thread(file_path = :missing)
```

Comment:
Given that this is part of the public interface, please add a short section in the README documented this method and the stop one as well. And be sure to mention that you may not start if already started.